benchmarks/profiler/deploy/profile_sla_aic_dgdr.yaml

```
@@ -12,7 +12,7 @@ spec:
 
   # ProfilingConfig maps directly to the profile_sla.py config format
   profilingConfig:
-    profilerImage: "nvcr.io/nvidian/dynamo-dev/vllm-runtime:dep-554.0"
+    profilerImage: "nvcr.io/nvidian/dynamo-dev/vllm-runtime:my-tag"
```

Do you mind changing these to `nvidia/ai-dynamo` as well?
```suggestion
    profilerImage: "nvcr.io/nvidia/ai-dynamo/vllm-runtime:my-tag"
```